Building `0.0.125` on `i686-linux-android` fails

Building version `0.0.125` on 32 bit fails with

> attempt to compute `184_usize - 192_usize`, which would overflow

```
cargo ndk -t i686-linux-android build --release
...

error[E0080]: evaluation of constant value failed
   --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/lightning-0.0.125/src/routing/scoring.rs:807:46
    |
807 | const _LIQUIDITY_MAP_SIZING_CHECK_2: usize = ::core::mem::size_of::<(u64, ChannelLiquidity)>() - 192;
    |                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ attempt to compute `184_usize - 192_usize`, which would overflow

For more information about this error, try `rustc --explain E0080`.
error: could not compile `lightning` (lib) due to 1 previous error
```

Maybe `usize` on 32 bit is too small to hold `_LIQUIDITY_MAP_SIZING_CHECK_2`?

https://github.com/lightningdevkit/rust-lightning/blob/8da30df223d50099c75ba8251615bd2026fcea75/lightning/src/routing/scoring.rs#L805

This worked fine with the previous version we used, `0.0.118`.

Is this a bug, or was support for 32 bit dropped in the meantime? Thanks.

--- a/lightning/src/routing/scoring.rs
+++ b/lightning/src/routing/scoring.rs
@@ -821,16 +821,6 @@ struct ChannelLiquidity {
 	offset_history_last_updated: Duration,
 }
 
-// Check that the liquidity HashMap's entries sit on round cache lines.
-//
-// Specifically, the first cache line will have the key, the liquidity offsets, and the total
-// points tracked in the historical tracker.
-//
-// The next two cache lines will have the historical points, which we only access last during
-// scoring, followed by the last_updated `Duration`s (which we do not need during scoring).
-const _LIQUIDITY_MAP_SIZING_CHECK: usize = 192 - ::core::mem::size_of::<(u64, ChannelLiquidity)>();
-const _LIQUIDITY_MAP_SIZING_CHECK_2: usize = ::core::mem::size_of::<(u64, ChannelLiquidity)>() - 192;
-
 /// A snapshot of [`ChannelLiquidity`] in one direction assuming a certain channel capacity.
 struct DirectedChannelLiquidity<L: Deref<Target = u64>, HT: Deref<Target = HistoricalLiquidityTracker>, T: Deref<Target = Duration>> {
 	min_liquidity_offset_msat: L,

